lib/shared/src/tracing/index.ts

```diff
@@ -0,0 +1,40 @@
+import opentelemetry, { SpanStatusCode } from '@opentelemetry/api'
+
+const TRACE_NAME = 'cody'
+const TRACE_VERSION = '0.1'
```

Comment:
To comply with OpenTelemetry [terminology](https://opentelemetry.io/docs/instrumentation/js/manual/#acquiring-a-tracer):

```suggestion
const INSTRUMENTATION_NAME = 'cody'
const INSTRUMENTATION_VERSION = '0.1'
```

OpenTelemetry contains a lot of abstractions, so it's helpful when our naming matches theirs when we need to find something in the spec or docs.